### PR TITLE
CT-573 remove deprecated jobs, workflows, and commands from circleci

### PR DESCRIPTION
**Pull request by dlobue**
Removes deprecated jobs, workflows and commands from circleci for benchmarks and smoke-tests.

CT-573

### CODE REVIEW

**Comment by ArthurKamalov** [comment]
All looks good to me except the fact that I tried to explain in this slack message https://sentinelone.slack.com/archives/C01S3UJTBLG/p1661195575221639?thread_ts=1661195222.549139&cid=C01S3UJTBLG

Maybe we should port at least that missing rate limiting tests first?

**Comment by dlobue** [comment]
> All looks good to me except the fact that I tried to explain in this slack message https://sentinelone.slack.com/archives/C01S3UJTBLG/p1661195575221639?thread_ts=1661195222.549139&cid=C01S3UJTBLG
> 
> Maybe we should port at least that missing rate limiting tests first?

They have been ported over: https://github.com/scalyr/scalyr-agent-2/blob/master/.github/workflows/unittests.yml#L140-L210
https://github.com/scalyr/scalyr-agent-2/actions/runs/2906419700
Here's the rate-limit test for this PR specifically: https://github.com/scalyr/scalyr-agent-2/runs/7959264643?check_suite_focus=true

**Comment by ArthurKamalov** [comment]
Ah, so they are in unittests. Sorry, my bad.

**Review by ArthurKamalov — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -91,352 +91,6 @@ parameters:
     default: ""
 
 commands:
-  unittest_tox:
```
Comment (Kami) [inline .circleci/config.yml:94]: Previously I only removed job names from the section at the bottom where workflows are defined (aka which jobs run for which workflow), but removing it all together from this file is fine as well (if we ever need to add it back for some reason, there is always git history :)).

**Pull request merged**